Dockerfile

```diff
@@ -6,6 +6,8 @@ ARG GITHUB_OAUTH_TOKEN=notset
 COPY . .
 
 RUN git describe --always --dirty --tags | tee VERSION
+RUN git rev-parse --short HEAD | tee HEROKU_SLUG_COMMIT
```

Comment:
I think it is a good idea to use a name that's not associated with Heroku at this time.

Reply:
@BanzaiMan I changed to HEROKU_SLUG_COMMIT